contracts/cw721-fixed-price/src/contract.rs

```diff
@@ -372,7 +370,7 @@ mod tests {
             res.messages[0],
             SubMsg {
                 msg: CosmosMsg::Wasm(WasmMsg::Execute {
-                    contract_addr: String::from(MOCK_CONTRACT_ADDR),
+                    contract_addr: "nftcontract".to_string(),
```

Comment:
best to keep using  MOCK_CONTRACT_ADDR as a variable

Reply:
Ah, but that's used for the cw20 address...

Reply:
Added a constant for this, should be good.